src/store/flow/utils/importHelpers.ts

```diff
@@ -27,21 +27,26 @@ export function updateResourcesForLanguageMatch(
 export function mergeFlowContainer(
   existingFlowContainer: IContext, newFlowContainer: IContext,
 ): IContext {
-  const newExistingFlowContainer = cloneDeep(existingFlowContainer)
-  // There should only be a single flow imported at a time currently.
-  // We block any more than this in the importer
-  const newFlow = get(newFlowContainer, 'flows[0]')
-  const newFlowUUID = get(newFlowContainer, 'flows[0].uuid')
-  const existingFlowIndex = findIndex(
-    newExistingFlowContainer.flows,
-    (flow) => flow.uuid === newFlowUUID,
-  )
-  if (existingFlowIndex < 0) {
-    newExistingFlowContainer.flows.push(newFlow)
-  } else {
-    newExistingFlowContainer.flows[existingFlowIndex] = newFlow
-  }
-  return newExistingFlowContainer
+  const existingFlowContainerStore = cloneDeep(existingFlowContainer);
+
+  const newFlows = get(newFlowContainer, 'flows', []);
+
+  newFlows.forEach((newFlow) => {
+    const newFlowUUID = get(newFlow, 'uuid');
```

Comment:
we probably don't need `get` because a flow always has a uuid.
maybe use `newFlow.uuid` + inline it?
(and the same for `newFlows` above)